lib/gitarro/git_op.rb

```diff
@@ -49,7 +49,7 @@ def del_pr_branch(upstream, pr)
   def ck_or_clone_git
     git_repo_dir = git_dir + '/' + @options[:repo].split('/')[1]
     return if File.directory?(git_repo_dir)
-    FileUtils.mkdir_p(git_dir) if File.directory?(git_dir)
+    FileUtils.mkdir_p(git_dir) unless File.directory?(git_dir)
```

Comment:
we need unlss, otherwise we check create the dir if the dir exist

Reply:
Oops, that's totally true @MalloZup! I didn't catch it! Thanks for fixing! 🚀